.vsts-ci.yml

```diff
@@ -11,7 +11,7 @@ variables:
 resources:
   containers:
   - container: ubuntu1404
-    image: lightgbm/vsts-agent:ubuntu-14.04
+    image: lightgbm/vsts-agent:ubuntu-14.04-dev
```

Comment:
@jameslamb 
> All working! @StrikerRUS could you take one more look?

👍  

https://github.com/guolinke/lightgbm-ci-docker/pull/7 has been merged, so you can switch container back to the production version.

Reply:
switched back in https://github.com/microsoft/LightGBM/pull/2530/commits/9ab619c955e96321c474cc5f37ffce6687d6c368